src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
```

Comment:
I would have no problem with not using `Token.toChars`.  It's not like we're going to change the names.